include/config.h

```diff
@@ -0,0 +1,40 @@
+#ifndef KANSHI_CONFIG_H
+#define KANSHI_CONFIG_H
+
+#include <wayland-client.h>
+
+enum kanshi_output_field {
+	KANSHI_OUTPUT_ENABLED = 1 << 0,
+	KANSHI_OUTPUT_MODE = 1 << 1,
+	KANSHI_OUTPUT_POSITION = 1 << 2,
+	KANSHI_OUTPUT_SCALE = 1 << 3,
+	KANSHI_OUTPUT_TRANSFORM = 1 << 4,
+};
+
+struct kanshi_profile_output {
+	char *name;
+	unsigned int fields; // enum kanshi_output_field
+	struct wl_list link;
+
+	bool enabled;
+	struct {
+		int width, height;
+		int refresh; // mHz
+	} mode;
+	struct {
+		int x, y;
+	} position;
+	float scale;
+	enum wl_output_transform transform;
```

Comment:
dpms field?

Reply:
This was intentionally left out of the output-management protocol (see https://github.com/swaywm/wlr-protocols/pull/38). I think DPMS should be handled by swayidle instead.